python/lsst/meas/algorithms/brightStarStamps.py

```diff
@@ -56,56 +55,62 @@ class BrightStarStamp(AbstractStamp):
     annularFlux : `Optional[float]`
```

Comment:
```suggestion
    annularFlux : `float`, optional
```

Reply:
These have two different meanings. The first is that `float` or `None` are allowed. The second is that you must give a `float` but things work if you don't give anything at all (the default value will be used).

Reply:
We'd want something like this then: https://github.com/lsst/ip_isr/blob/4b253454a05df26a7ddd28689a0876f006854e98/python/lsst/ip/isr/vignette.py#L87

Reply:
Thanks, changed here (and elsewhere that I caught it).